check_docker.py

```diff
@@ -224,9 +224,10 @@ def process_args(args):
 
     # Container name
     parser.add_argument('--containers',
+                        required=True,
```

Comment:
Hmm, I would argue it makes more sense for "all" to be explicit, but I honestly didn't realize it used it by default, largely due to the lack of a default test at all. I'll remove the requirement.